tap_klaviyo/__init__.py

```diff
@@ -76,7 +84,21 @@ def to_catalog_dict(self):
     'full'
 )
 
-FULL_STREAMS = [GLOBAL_EXCLUSIONS, LISTS, LIST_MEMBERS]
+EVENTS = Stream(
+    'events',
+    'events',
+    'uuid',
+    'full'
+)
+
+PROFILES = Stream(
+    'profiles',
+    'profiles',
+    'id',
+    'full'
+)
+
+FULL_STREAMS = [GLOBAL_EXCLUSIONS, LISTS, LIST_MEMBERS, EVENTS, PROFILES]
```

Comment:
If I remember correctly, `FULL_STREAMS` refers to streams extracted in a full overwrite manner. Is that right? Or is it just the full list of streams? I don't think we should extract things like `EVENTS` full overwrite -- can we not do it incrementally?

Reply:
This came from [here](https://github.com/hotgluexyz/tap-klaviyo/blob/master/tap_klaviyo/__init__.py#L97C1-L97C73) and as far as I can tell, `FULL_STREAMS` is only used in the `discover` operation, so I don't think it actually drives anything related to full/incremental.

Yeah this code is really something.